src/index.tsx

```diff
@@ -29,19 +31,36 @@ export interface IReactToPrintProps {
     removeAfterPrint?: boolean;
     /** Optional - suppress error messages */
     suppressErrors?: boolean;
+    /** Optional - download pdf file directly */
+    downloadPDF?: boolean;
+    /** Optional - document title for PDF file */
+    documentTitle?: string;
```

Comment:
What is the title of the PDF if this is not passed? Should there be a default value to the prop?